gssapi/tests/test_raw.py

```diff
@@ -28,6 +33,45 @@ def setUpClass(cls):
 
         cls.USER_PRINC = cls.realm.user_princ.split('@')[0].encode("UTF-8")
         cls.ADMIN_PRINC = cls.realm.admin_princ.split('@')[0].encode("UTF-8")
+        cls.KRB5_LIB_PATH = cls._get_krb5_lib_path()
+
+    @classmethod
+    def _get_krb5_lib_path(cls) -> str:
```

Comment:
Yea I couldn't figure out a way to avoid this. I thought at least having it tested in CI is better than no tests at all but if you prefer not to have this logic and just skip that is ok.

Another thought is to just read the value from an env var and skip it if it's not defined. This can allow the test suite to run normally without these tests locally and CI can have it set as we would know the correct paths beforehand. What do you think of that?

Reply:
Oh, good point about `gss_krb5_copy_ccache()`.  Will think about the rest.

Reply:
I'll set the env var as it is definitely safer and at least keeps the tests running in CI. If you think of another way or want to do something different then let me know.

Reply:
The latest commit adds this and sets the env var in the distro specific section of `ci/lib-setup.sh`.